lib/integrations/slack/send_on_slack_service.rb

```diff
@@ -117,6 +117,8 @@ def post_message
   end
 
   def upload_file
+    return if message.attachments.first.file.blank?
```

Comment:
I think the issue is that the attachment is a contact, location, share, or fallback, and there’s no actual file to attach.

Instead of checking for the presence of a file, should we just check for the type of attachment?

Here’s an example of what I mean.

```rb
def valid_slack_attachment?(attachment)
  [:image, :audio, :video, :file].includes(attachment.file_type)
end

return unless valid_slack_attachment?(message.attachments.first)
```